From b7cb02f4d17a5c9764681dd937dbb49e721a9baa Mon Sep 17 00:00:00 2001
From: Ryan QIAN <b32804@freescale.com>
Date: Thu, 8 Dec 2011 13:38:33 +0800
Subject: [PATCH 0705/2463] ENGR00169782 [MX6Q]: Add new AR6003 driver to
 2.6.38, replacing staging one

- remove the staging driver from default config of MX6

Signed-off-by: Ryan QIAN <b32804@freescale.com>
---
 arch/arm/configs/imx6_defconfig |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index f7ffa37..5ac6c16 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -1857,7 +1857,7 @@ CONFIG_DMA_ENGINE=y
 # CONFIG_DMATEST is not set
 # CONFIG_AUXDISPLAY is not set
 # CONFIG_UIO is not set
-CONFIG_STAGING=y
+# CONFIG_STAGING is not set
 # CONFIG_STAGING_EXCLUDE_BUILD is not set
 # CONFIG_USB_DABUSB is not set
 # CONFIG_USB_SE401 is not set
@@ -1885,8 +1885,8 @@ CONFIG_STAGING=y
 # CONFIG_SMB_FS is not set
 # CONFIG_EASYCAP is not set
 CONFIG_MACH_NO_WESTBRIDGE=y
-CONFIG_ATH6K_LEGACY=m
-CONFIG_AR600x_SD31_XXX=y
+# CONFIG_ATH6K_LEGACY is not set
+# CONFIG_AR600x_SD31_XXX is not set
 # CONFIG_AR600x_WB31_XXX is not set
 # CONFIG_AR600x_SD32_XXX is not set
 # CONFIG_AR600x_CUSTOM_XXX is not set
-- 
1.7.7.4

